keyboards/bm16a/keymaps/stock/readme.md

```diff
@@ -0,0 +1,4 @@
+# The default keymap for bm16a
```

Comment:
Might want to change the readme or remove it

Reply:
I was merely following the readme in the default keymap. If you dislike it then I'll gladly delete it, but then users might fail to understand the reason for this keymap's existence.

Reply:
Maybe a

```diff
diff --git a/keyboards/bm16a/keymaps/factory/readme.md b/keyboards/bm16a/keymaps/factory/readme.md
index b5dc5e731..148cf29af 100644
--- a/keyboards/bm16a/keymaps/factory/readme.md
+++ b/keyboards/bm16a/keymaps/factory/readme.md
@@ -1,4 +1,4 @@
-# The default keymap for bm16a
+# The factory keymap for bm16a

-This is the default layout this keypad ships with. Use it if you want to return to stock.
+This is the layout this keypad ships with. Use it if you want to return to stock.
```

and a 

```
git mv -- keyboards/bm16a/keymaps/stock/ keyboards/bm16a/keymaps/factory/
```

?

Reply:
Its more "stock" like, as we have no audit trail of what the board actually shipped with. Returning to stock would likely involve dumping some fimware from a non QMK flashed board, or begging kprepublic for the original fimware.

Reply:
Thank you @noroadsleft. I initially named the keymap "factory", but later changed it to stock. I reverted it to be what it was.

Those are some valid points @zvecr. I changed the readme accordingly.